.github/workflows/request-reviews.yml

```diff
@@ -35,62 +35,28 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0
+          fetch-depth: 1
+          sparse-checkout: |
+            .github
+            BaseTools/Scripts

```

Comment:
Why BaseTools/Scripts?  I see references to files in .github/scripts below.

Reply:
Good catch. We shouldn't need that anymore after I moved the scripts from BaseTools to .github. I'll confirm and drop it.

Reply:
I forgot this is needed to access BaseTools/Scripts/GetMaintainer.py.

Reply:
Can you add the list of file/dirs in the PR summary in item 3 so it is clear why those specific sparse checkout rules are being used.  And/Or a comment in the YML file.

Reply:
Sure

Reply:
Comments added in latest push.